### ISSUE TITLE
Add Control Flow Graph visualization to the playground.

### ISSUE THREAD

**Issue by rzvxa** [oxc-project/oxc#3423]
I hope we can adopt [this](https://github.com/dreampuf/GraphvizOnline) project for this purpose, It's been my go-to visualizer, and having it in the playground would be a nice to have feature.

I've self-assigned it but if anyone decides to pick it up sooner feel free to do so, and announce it in this issue before starting to work on it.

**Comment by Boshen** [comment]
We can build this in the new playground https://github.com/oxc-project/playground
